lighthouse-core/test/audits/image-aspect-ratio-test.js

```diff
@@ -32,7 +32,7 @@ describe('Images: aspect-ratio audit', () => {
           generateImage(
             {width: data.clientSize[0], height: data.clientSize[1]},
             {naturalWidth: data.naturalSize[0], naturalHeight: data.naturalSize[1]},
-            generateRecord(),
+            generateRecord({mimeType: data.mimeType}),
```

Comment:
it's expecting two parameters so I believe this should look like `generateRecord(undefined, data.mimeType)`

then again, it doesn't really look like `generateRecord` is very useful for this test at all and was copied from another file so you can probably just ditch the function entirely and pass in the object directly to `generateImage`